release_notes/2.7.0.md

```diff
@@ -0,0 +1,5 @@
+* Added the 'delete event' action
+* Added 'download email' parameter in the 'get email' action
+* Updated the 'get email' action to provide event related information
```

Comment:
Should we include ticket numbers for these bullet points as well? (e.g. [PAPP-1234])

Reply:
Also please make sure to follow the suggested merge order described in the README of this PR. Thanks!

Reply:
@paulnawat, As this is a community-submitted PR, we don't add ticket numbers in the release notes. Also, all the dependencies are merged.